package.json

```diff
@@ -40,7 +40,9 @@
     "forward-time": "node scripts/forward-time",
     "playwright:install": "npx playwright install --with-deps",   
     "playwright:run": "playwright test",
-    "playwright:watch": "playwright test --ui"
+    "playwright:watch": "playwright test --ui",
+    "watch:lambdas": "node scripts/transpile-lambdas.js",
```

Comment:
This file `scripts/transpile-lambdas.js` doesn't exist?

Reply:
Thank you! Left over from previous implementation 🙃

Reply:
I'll be leaving it in after all after reverting back to a previous implementation 😄